lib/features/family/features/auth/pages/login_page.dart

```diff
@@ -0,0 +1,256 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app/features/auth/pages/change_password_page.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/shared/dialogs/dialogs.dart';
+import 'package:givt_app/shared/widgets/buttons/givt_elevated_button.dart';
+import 'package:givt_app/shared/widgets/widgets.dart';
+import 'package:givt_app/utils/util.dart';
+import 'package:go_router/go_router.dart';
+
+class LoginPage extends StatefulWidget {
```

Comment:
I suggest renaming it to USLoginPage or something that at least indicates that its the family app loginpage.